src/chapters/chapter_1/induction.tex

```diff
@@ -15,23 +14,12 @@ \subsection{Prove that $\sum_{i=1}^{n} i = n(n+1)/2$ for $n \geq 0$, by inductio
            &= 1
     \end{align}
 
-    Thus, the inductive step is true.
+    Thus, the base step is true.
 
     \textbf{Inductive step}
-    If all of $P(1) + ... + P(n)$ are true, then $P(n+1)$ is true. Let's try to
-    get the value of $P(n+1)$.
+    Assume all of $P(1), ..., P(n)$ are true. Then we show that $P(n+1)$ is true as well:
```

Comment:
You cannot add P(1) + ... + P(n) as they are propositions. E.g. P(1) tells you \sum_1^1 i = 1*(1+1)/2 is true. P(2) tells you \sum_1^2 i = 2*(2+1)/2 is true.